skycatalogs/objects/base_object.py

```diff
@@ -70,7 +74,7 @@ def load_lsst_bandpasses():
         bp = bp.withZeropoint('AB')
         lsst_bandpasses[band] = bp
 
-    return lsst_bandpasses
+    return lsst_bandpasses, version
```

Comment:
True.  load_lsst_bandpasses could instead store the version as well as the bandpasses in a separate global variable and there could be a separate get_lsst_bandpasses_version routine.  It would be neater to do this with a class rather than adding more global variables, but that also would break the interface.

Reply:
I suggest an internal version of the function in addition to the public interface.  So change the name of the current `load_lsst_bandpasses` to `_load_lsst_bandpasses`, and make a new public function that just returns the dict, so something along the lines of
```
def load_lsst_bandpasses():
    return _load_lsst_bandpasses()[0]
```